osp/utils/zulip_api.py

```diff
@@ -35,7 +63,16 @@ def get_newest_message(zulip_id):
     return result
 
 
-def get_stream_messages(stream, zulip_id):
+def get_stream_messages(stream: str, zulip_id: int) -> int:
+    """It takes stream name: string and target id: int as arguements
+    and returns the total number of messages sent on the given stream: int, by user associated with given user id
+
+    Example usage:
+
+            >>> get_stream_messages(announce,5)
+            10
+    """
+
     request = {
         "anchor": "newest",
         "num_before": 1,
```

Comment:
Also, @isabelcosta this `num_before` value must be increased to 1000 to get correct stats for stream messages.

Reply:
really? issue please 🙏🏾 i don't entirely understand your point here, but with an issue I might

Reply:
Ok, I will make an issue @isabelcosta